src/ST_DeviceTypes.js

```diff
@@ -733,11 +733,6 @@ module.exports = class DeviceTypes {
         thisChar = accessory
             .getOrAddService(Service.Thermostat)
             .getCharacteristic(Characteristic.CurrentTemperature)
-            .setProps({
```

Comment:
Out of my own curiosity can you explain why you think these should be removed.

Reply:
I believe these are set to default values for a characteristic when adding a thermostat. Additionally they were set to very high values, making my thermostat practically useless as they were out of the device's permitted range.